api/ask_astro/chains/answer_question.py

```diff
@@ -12,9 +12,10 @@
     SystemMessagePromptTemplate,
 )
 from langchain.retrievers import MultiQueryRetriever
+from langchain.retrievers.weaviate_hybrid_search import WeaviateHybridSearchRetriever
 
-from ask_astro.clients.weaviate_ import docsearch
```

Comment:
is this now dead code?

https://github.com/astronomer/ask-astro/blob/hybrid-search/api/ask_astro/clients/weaviate_.py#L29-L35

Reply:
Yes